src/main/java/org/commonjava/service/storage/config/StorageServiceConfig.java

```diff
@@ -35,4 +40,12 @@ public interface StorageServiceConfig
 
     @WithName( "removableFilesystemPattern" )
     String removableFilesystemPattern();
+
+    @WithName( "type" )
+    @WithDefault( "" )
```

Comment:
maybe just set the default to nfs, and manually update the config for mp+ , so it won't affect current environments.

Reply:
all others look good to me.

Reply:
updated